tools/KoreBuild.Console/Commands/DockerFiles/winservercore.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/aspnet:4.6.2
+
+ADD https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe /net461dev.exe
+
+RUN /net461dev.exe /Passive /NoRestart
+
+RUN rm /net461dev.exe 
+
+ADD . c:/repo/
```

Comment:
I decided not to dump this in c:\ because it's cleaner, but also because I ran into a problem with running KoreBuild from the drive root on windows.

Reply:
I've typically seen Dockerfiles use the following order of statements instead:

```
WORKDIR c:/repo/

ADD . .
```

Reply:
Also, our docs (https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile) recommend escaped backslashes rather than forward slashes, though both may work:

```
WORKDIR c:\\windows
```